tf_encrypted/tensor/int100.py

```diff
@@ -568,6 +568,10 @@ def strided_slice(self, args, kwargs):
             backing = [tf.strided_slice(xi, *args, **kwargs) for xi in self.backing]
             return DenseTensor(backing)
 
+        def gather(self, indices: list, axis: int = 0):
+            backings = [tf.gather(xi, indices, axis=axis) for xi in self.backing]
+            return [DenseTensor(backing) for backing in backings]
```

Comment:
I'm not sure why you want to return a list here? Shouldn't it just be a single tensor?

Reply:
ah good catch, I had fixe it in `native` but not in int100. thanks!